src/App.tsx

```diff
@@ -6,67 +6,26 @@ import './App.scss';
 import ArtView from './ArtView/ArtView';
 import {Route, Switch} from 'react-router-dom';
 
-
 interface State {
-  artPieces: Props[];
-  imgId: string;
-  singleImageDetails: {
-    date_end: number;
-    date_start?: number;
-    image_id?: string;
-    title?: string;
-    _score?: number;
-  } | object | undefined
-}
-
-interface Props {
-  id: number;
-  image_src: string;
-  description: string;
-  title: string;
-  date_start: number;
-  image_id: string;
-  artist_title: string;
-  classification_titles: Array<string>;
+  categories: Array<string>;
 }
 
 class App extends Component<{}, State> {
   constructor(props: object) {
     super(props)
+
     this.state = {
```

Comment:
Awesome job cleaning out this app, and re-arranging the interfaces to reflect the new state in App.

Reply:
Agree, this looks great!